spikes/spine-adopt/wazero-probe/report.go

```
@@ -0,0 +1,644 @@
+			t.Match++
+		case statusExplained:
+			t.Explained++
+		case statusUnexplained:
+			t.Unexplained++
+		}
+	}
+	return t
+}
+
+func writeArtifact(dir, name string, v any) (string, error) {
+	if err := os.MkdirAll(dir, 0o755); err != nil {
```

golangci-lint (2.12.2)

[medium] G301: Expect directory permissions to be 0750 or less

(gosec)